FrontEnd/src/components/HeaderFooter/header/navbar/Logout.jsx

```diff
@@ -1,23 +1,27 @@
 import axios from 'axios';
 import { useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
 import { useAuth } from '../../../../hooks';
 
 function Logout() {
   const auth = useAuth();
   const navigate = useNavigate();
 
-  const onClick = async () => {
-    await axios.post(
-      `${process.env.REACT_APP_BASE_API_URL}/api/auth/token/logout`
-    );
-    auth.logout();
-    navigate('/');
-  };
-  return (
-    <button onClick={onClick} type="submit">
-      Вихід
-    </button>
-  );
+  useEffect(() => {
+    const performLogout = async () => {
+      if (auth.isAuth) {
+        try {
+          await axios.post(`${process.env.REACT_APP_BASE_API_URL}/api/auth/token/logout`);
+          await auth.logout();
```

Comment:
Could we move POST request into `auth.logout`?

Reply:
I'll try to do it as a part of task on updating logout functionality ( #394 in progress now). This post request is not only in Logout component, but in DeleteProfileModal too.